src/components/application_manager/src/commands/hmi/sdl_activate_app_request.cc

```diff
@@ -58,6 +58,7 @@ void SDLActivateAppRequest::Run() {
     LOG4CXX_ERROR(
         logger_,
         "Can't find application within regular apps: " << application_id);
+    SendResponse(false, correlation_id(), SDL_ActivateApp, APPLICATION_NOT_REGISTERED);
```

Comment:
@LuxoftAKutsan , do you know, that according to req-t you mustn't immediately send response. We have to wait time-out and then send response. And in line 106 we have already one implementation of it.  See [APPLINK-24453](https://adc.luxoft.com/jira/browse/APPLINK-24453).